### ISSUE TITLE
emit is not working as it is supposed to

### ISSUE THREAD

**Issue by siddo420** [primus/omega-supreme#8]
server:

```js
primus.forward.sparks([spark.id], {
  event: 'tonk',
  data: { name: 'sid' }
}, function (err, results) {
  Logger.infoc('In Tonk event: ' + JSON.stringify( results ));
});
```

on client:

```js
primus.on('tonk', function (data) {
  console.error('Data on tonk: ' + data );
});

primus.on('data', function (data) { ... });
```
- In my opinion, the tonk event handler should be called on client but instead the data event gets called. Is this known and intended behaviour?
- Do I have to generate the event on client manually?
- If I understood it correctly, will it (event generation) work on multiple sparks connected from separate clients (multiple hosts)?

**Comment by lpinca** [comment]
The documentation is wrong, I'll fix it later.
If you are using `primus-emit` use the following format to fire custom events:

```js
primus.forward('http://localhost:8080', {
  emit: [ 'eventname', 'firstarg', 'secondarg' ]
}, fn);
```

**Comment by siddo420** [comment]
Seems to work now, please correct the docs.

Also, before I test it, will it (event generation) work on multiple sparks connected thru separate clients (multiple hosts)?

**Comment by lpinca** [comment]
Yes, it should. I've already fixed the doc in fbc104520614fc586222235816858d169b8748c3.
